conans/conan.py

```diff
@@ -1,5 +1,6 @@
 import sys
-
+import multiprocessing
+multiprocessing.freeze_support()
```

Comment:
mmm, why not inside `if __name__ == '__main__':` like is documented in https://docs.python.org/3/library/multiprocessing.html#multiprocessing.freeze_support?? 

Unless pyinstaller advises differently?